xikolo-ios/ViewControllers/Courses/CertificatesListViewController.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  Created for xikolo-ios under MIT license.
+//  Copyright © HPI. All rights reserved.
+//
+
+import DZNEmptyDataSet
+import SafariServices
+import UIKit
+
+class CertificatesListViewController: UITableViewController {
+
+    var course: Course!
+
+    var certificates: [(String, URL?)] = []
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        certificates = findAvailableCertificates()
```

Comment:
This should be done in `viewDidAppear` since the user can go to this tab (certificate not achieved yet), than visit show items and come back (certificate now achieved). We should also refresh the enrollment beforehand.